Update HCOMisconfiguredDescheduler runbook (#322)
kubevirt/monitoring: docs/runbooks/HCOMisconfiguredDescheduler.md

The order of preference is:

1. `KubeVirtRelieveAndMigrate`  
2. `DevKubeVirtRelieveAndMigrate`  
3. `LongLifecycle` with the `devEnableEvictionsInBackground` customization  

You can check which profiles are supported in your descheduler release with:

```bash
oc get crd kubedeschedulers.operator.openshift.io -o json | jq '.spec.versions[] | select(.name=="v1").schema.openAPIV3Schema.properties.spec.properties.profiles.items.enum[]'
```

Depending on the available profiles, apply one of the following configurations
to the `KubeDescheduler` CR (kind: `kubedeschedulers`, namespace:
`openshift-kube-descheduler-operator`, name: `cluster`):

* **Configure the `KubeVirtRelieveAndMigrate` profile**  
   ```yaml
   spec:
     profiles:
     - KubeVirtRelieveAndMigrate
   ```

* **Configure the `DevKubeVirtRelieveAndMigrate` profile**  
   ```yaml
   spec:
     profiles:
     - DevKubeVirtRelieveAndMigrate
   ```

* **Configure the `LongLifecycle` profile with the
`devEnableEvictionsInBackground` profile customization**  
   ```yaml
   spec:
     profiles:
     - LongLifecycle
     profileCustomizations:
       devEnableEvictionsInBackground: true
   ```

> * **Remove the `Kube Descheduler Operator`** if none of the above is feasible.

Reviewer: ```suggestion
If none of the listed steps are possible, **remove the `Kube Descheduler Operator`**.
```
